### PR TITLE
[Ready for Review + Merge] - Phase 1 - Trail Photos DB Seed + API

### PR DESCRIPTION
**Pull request by MengRS**

### CODE REVIEW

**Comment by MengRS** [comment]
@ryanbrennan12 - assigning you as primary reviewer
@weber93 - assigning you as secondary (back-up) reviewer

File: package.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
     "startWeb": "nodemon server/index.js",
     "openPage": "open http://localhost:3003",
     "test": "jest",
-    "loadData": "sed -i '' '16,$ d' database-postgresql/schema.sql",
-    "seedData": "npm run loadData; node database-postgresql/seed.js; psql trailPhotosDB < database-postgresql/schema.sql"
+    "loadData": "psql trailPhotosDB < database-postgresql/schema.sql",
+    "seedData": "sed -i '' '16,$ d'; node database-postgresql/seed.js; npm run loadData"
```
Comment (rybrennan) [inline package.json:17]: per discussion on start script `"start": "nodemon server/index.js | npm run react-dev | npm run openPage"`

Comment (MengRS) [reply package.json:17]: @ryanbrennan12, this should be fixed - please repull and lmk - thanks!

Comment (rybrennan) [reply package.json:17]: all good

**Pull request merged**